test/test_images/probe_helper/probe_helper.go

```diff
@@ -143,7 +147,7 @@ func logReceive(ctx context.Context, ack bool, format string, args ...interface{
 	return cloudevents.NewReceipt(ack, format, args...)
 }
 
-func (ph *ProbeHelper) forwardFromProbe(ctx context.Context, brokerClient cloudevents.Client, pubsubClient cloudevents.Client, bucket *storage.BucketHandle, receivedEvents *receivedEventsMap) cloudEventsFunc {
+func (ph *ProbeHelper) forwardFromProbe(ctx context.Context, brokerClient cloudevents.Client, cePubsubClient cloudevents.Client, pubsubClient *pubsub.Client, bucket *storage.BucketHandle, receivedEvents *receivedEventsMap) cloudEventsFunc {
```

Comment:
nit: the clients can be moved to the ProbeHelper struct

Reply:
I moved lots of things to the ProbeHelper struct and allow them to be set manually if desired.